### PR TITLE
Dogma 0.5.0 identity

### PR DESCRIPTION
**Pull request by koden-km**
This is the `TimeoutHint()` and `Identity()` part of the dogma api upgrade.

Refs #15

### CODE REVIEW

File: config/aggregate_test.go [hunk 1]
```diff
@@ -38,11 +38,17 @@ var _ = Describe("type AggregateConfig", func() {
 			})
 
 			Describe("func Name()", func() {
-				It("returns the handler name", func() {
+				It("returns the name", func() {
```
Comment (jmalloc) [inline config/aggregate_test.go]: This can be reverted.

File: config/integration_test.go [hunk 2]
```diff
@@ -38,11 +38,17 @@ var _ = Describe("type IntegrationConfig", func() {
 			})
 
 			Describe("func Name()", func() {
-				It("returns the handler name", func() {
+				It("returns the name", func() {
```
Comment (jmalloc) [inline config/integration_test.go]: This can be reverted.

File: config/projection_test.go [hunk 3]
```diff
@@ -40,12 +40,22 @@ var _ = Describe("type ProjectionConfig", func() {
 				Expect(cfg.HandlerName).To(Equal("<name>"))
 			})
 
+			It("the handler key is set", func() {
```
Comment (jmalloc) [inline config/projection_test.go]: I think this test might be at the wrong level or something, but either way "It the handler key is set" does not read like plain English.

File: config/process_test.go [hunk 4]
```diff
@@ -42,12 +42,22 @@ var _ = Describe("type ProcessConfig", func() {
 				Expect(cfg.HandlerName).To(Equal("<name>"))
 			})
 
+			It("the handler key is set", func() {
```
Comment (jmalloc) [inline config/process_test.go]: I think this test might be at the wrong level or something, but either way "It the handler key is set" does not read like plain English.

File: config/projection_test.go (hunk 3 above)
Comment (koden-km) [reply config/projection_test.go]: The "name" version of that test above is worded like that also.

File: config/aggregate_test.go (hunk 1 above)
Comment (koden-km) [reply config/aggregate_test.go]: I forgot to push this revert.

File: config/integration_test.go (hunk 2 above)
Comment (koden-km) [reply config/integration_test.go]: Thats the other one i was looking for.

File: config/projection_test.go (hunk 3 above)
Comment (jmalloc) [reply config/projection_test.go]: Ok, leave for now I guess.

Comment (koden-km) [reply config/projection_test.go]: I guess it needs a `Describe` wrapped around them. But these to are kind of odd tests to have, its more so making sure that `Identity()` was called?

What about something like this?
`It("ensures that the handler key is set", func() {`

Comment (jmalloc) [reply config/projection_test.go]: Probably just `It("sets the handler key")` - but now I feel like all those name ones I told you to revert need fixing too :/

Comment (koden-km) [reply config/projection_test.go]: Make a call and i'll do it. Lets get this done and move on.

Comment (koden-km) [reply config/projection_test.go]: As discussed, these are redundant and will be deleted.

File: config/process_test.go (hunk 4 above)
Comment (koden-km) [reply config/process_test.go]: As discussed, these are redundant and will be deleted.

**Pull request merged**